frontend/src/packages/trucks/trucks-api.ts

```diff
@@ -30,7 +34,27 @@ class TruckApi extends HttpApi {
       },
     );
 
-    return await response.json<TruckEntity>();
+    return await response.json<TruckEntityT>();
+  }
+
+  public async getAllTrucksByUserId({
+    userId,
+  }: Pick<
+    UsersTrucksEntityT,
+    'userId'
+  >): Promise<GetAllTrucksByUserIdResponseDto> {
+    const response = await this.load(
+      this.getFullEndpoint(TruckApiPath.$USER_ID, {
+        userId: userId.toString(),
+      }),
+      {
+        method: 'GET',
+        contentType: ContentType.JSON,
+        hasAuth: false,
```

Comment:
```suggestion
  public async getAllTrucksByUserId({
    userId,
  }: Pick<
    UsersTrucksEntityT,
    'userId'
  >): Promise<GetAllTrucksByUserIdResponseDto> {
    const response = await this.load(
      this.getFullEndpoint(TruckApiPath.$USER_ID, {
        userId: userId.toString(),
      }),
      {
        method: 'GET',
        contentType: ContentType.JSON,
        hasAuth: true,
```
there should be auth 🤔